index.js

```diff
@@ -115,24 +108,35 @@ function processBlock(block) {
             .then(function(svg) {
                 return book.output.writeFile(imgFilename, svg);
             });
-        })
-        .then(function() {
-            return {
-                isSVG:    true,
-                filename: imgFilename,
-                inline:   isInline
-            };
-        });
-    });
+        }
+    };
+}
+
+/**
+    Return assets for website
+
+    @return {Object}
+*/
+function getWebsiteAssets() {
+    var version = this.config.get('pluginsConfig.mathjax.version', '2.6.1');
```

Comment:
Yes, it's possible to set which is used by setting paramatere in book.json. I used this as default only because current archives use 2.6.1.

Reply:
That makes sense, and maybe should be a separate PR. I mention it because one should probably be using the latest version to avoid rendering and browser issues.

Reply:
If you want, I can make default version to 2.7.5 which is currently newest.
And in archive repo is already PR for using this as dependency.

Reply:
I'm not an authorized reviewer here, just pointing it out for them. You should probably leave it until one of them decides the proper course.

Reply:
I would just leave it like this. It's version currently used and this should be used just by archive to replace dependency on original repository. As I wrote, version can be changed up/down via configuration so there should be no problem :)